src/os/source/vectorsource.js

```diff
@@ -3217,6 +3234,29 @@ os.source.Vector.prototype.invertSelection = function() {
 };
 
 
+/**
+ * Convenience method select by id/array of ids
+ * @param {number|Array<number>} ids
+ * @param {boolean=} opt_deselect
+ * @suppress {accessControls}
+ */
+os.source.Vector.prototype.selectById = function(ids, opt_deselect) {
+  if (ids && !Array.isArray(ids)) {
```

Comment:
Instead of duplicating logic here and again in `hideById`, we may want another convenience function like `idsToFeatures`.